cmd/apiserver/app/server/run_server.go

```diff
@@ -42,7 +42,8 @@ func RunServer(opts *ServiceCatalogServerOptions) error {
 func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	tprOpts := opts.TPROptions
 	glog.Infoln("Installing TPR types to the cluster")
-	if err := tpr.InstallTypes(tprOpts.clIface); err != nil {
+	tprInstaller := tpr.NewInstallerTPRs(tprOpts.clIface)
```

Comment:
2 things:

1. Why do you need to create a new "object" (`struct`) here? Looks like it only has `InstallTypes` defined on it, so can we keep things simpler and just have a `func InstallTypes`? This is a repetition of what @MHBauer said [below](https://github.com/kubernetes-incubator/service-catalog/pull/466/files#r104093134)
2. Can you pass just the `ThirdPartyResources` interface to this function? A smaller interface will make this logic far easier to test